### PR TITLE
[PPR-345] Rule R7

### PR DESCRIPTION
**Pull request by aomegax**
#### List of Changes
- add check to create bundle request
- unit tests

#### Motivation and Context
A bundle subscription request is acceptable if and only if the public bundle is not expired.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/test/java/it/pagopa/afm/marketplacebe/TestUtil.java [hunk 1]
```diff
@@ -22,6 +24,7 @@ public static String getMockIdPsp() {
     public static String getMockCiFiscalCode() {
         return "fiscalCode";
     }
+    public static String getMockIdBundle() { return "cbfbc9c6-6c0b-429e-83ca-30ef453504f8"; }
```
Comment (jacopocarlini) [inline src/test/java/it/pagopa/afm/marketplacebe/TestUtil.java:27]:
```suggestion
    public final static String MOCK_ID_BUNDLE = "cbfbc9c6-6c0b-429e-83ca-30ef453504f8"; }
```
better?

File: src/main/java/it/pagopa/afm/marketplacebe/service/BundleRequestService.java [hunk 2]
```diff
@@ -201,4 +202,19 @@ private CiBundle buildCiBundle(BundleRequest entity) {
...
                 .build();
     }
+
+
+    /**
+     * Verify if validityDateTo is after now
+     * @param validityDateTo
+     * @return
+     */
+    private boolean isValidityDateToAcceptable(LocalDate validityDateTo) {
```
Comment (jacopocarlini) [inline src/main/java/it/pagopa/afm/marketplacebe/service/BundleRequestService.java:212]: move it in `CommonUtils`

**Review by jacopocarlini — APPROVED** [review]
LGTM with some comments

File: src/main/java/it/pagopa/afm/marketplacebe/service/BundleRequestService.java (hunk 2 above)
Comment (aomegax) [reply src/main/java/it/pagopa/afm/marketplacebe/service/BundleRequestService.java]: done 👍

**Review by jacopocarlini — DISMISSED** [review]
ok 👍

**Review by jacopocarlini — APPROVED** [review]

**Pull request merged**